vmdb/spec/controllers/ops_controller/diagnostics_spec.rb

```diff
@@ -126,5 +126,17 @@
         controller.send(:log_depot_edit)
       end
     end
+
+    context "#tree_select" do
+      it "renders zone list for root node" do
+        controller.instance_variable_set(:@_response, ActionController::TestResponse.new)
+        controller.instance_variable_set(:@sb, :active_tree => :diagnostics_tree)
+        controller.instance_variable_set(:@layout, 'ops')
+        controller.instance_variable_set(:@_params, :id => 'root', :action => 'tree_select')
+        controller.send(:tree_select)
```

Comment:
Thanks @tenderlove :+1: I will do as you wrote.

So generally you suggest to really fire the `post` and mock or seed whatever is needed, rather then just try to call a controller method, right?